Cabal/Distribution/Types/Component.hs

```diff
@@ -62,6 +63,14 @@ componentBuildInfo :: Component -> BuildInfo
 componentBuildInfo =
   foldComponent libBuildInfo foreignLibBuildInfo buildInfo testBuildInfo benchmarkBuildInfo
 
+lensBuildInfo :: (BuildInfo -> BuildInfo) -> (Component -> Component)
```

Comment:
We now depend on some Lenz stuff so maybe this can go elsewhere / use type synonym in the type?

Reply:
Yes, we have: https://github.com/haskell/cabal/blob/736c6f421488050893eb6efa55c76e7ba17d346e/Cabal/Distribution/Types/Component.hs#L41